boefjes/boefjes/plugins/kat_masscan/boefje.json

```diff
@@ -0,0 +1,18 @@
+{
+    "id": "masscan",
+    "name": "masscan",
+    "description": "Quickly scan large amounts of IPs.",
+    "consumes": [
+        "IPV4NetBlock"
+    ],
+    "produces": [
+        "IPAddressV4",
+        "IPAddressV6",
+        "IPPort"
+    ],
+    "environment_keys": [
+        "PORTS",
+        "MAX_RATE"
+    ],
+    "scan_level": 3
```

Comment:
This can be scan level 2. Nmap IP range is also level 2.

Reply:
I would not identify this tool as ‘touch at normal user level’ (https://docs.openkat.nl/introduction/howdoesitwork.html#indemnification). Maybe Nmap IP range needs a re-evaluation too. Potentially, the level should be made dependent on the configuration of the tool (I can imagine a difference between the top 10 ports and all 65535 ports).